packages/statistics-module/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@platzily/statistics-module",
+  "version": "1.0.0",
+  "description": "Statistics Services",
+  "author": "Statistics Team <@platzily/statistics>",
+  "homepage": "https://github.com/platzily/platzily-monolith#readme",
+  "license": "ISC",
```

Comment:
ISC?

Reply:
It is the same licence that l`ink-module` has :grimacing: